biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -0,0 +1,1172 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Working with Pandas\n",
+    "================="
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"overview-this-is-a-title overview\">\n",
+    "<h2 class=\"overview-title\">Overview</h2>\n",
+    "    \n",
+    "<p>Questions</p>\n",
+    "    <ul>\n",
+    "        <li>How can I use import data for analysis in my notebook?\n",
+    "    </ul>\n",
+    "<p>Objectives:</p>\n",
+    "    <ul>\n",
+    "        <li>Import the pandas library.\n",
+    "        <li>Use pandas library funtions to import data from a csv file.\n",
+    "        <li>Import data from a .csv formatted file.\n",
+    "    </ul>\n",
+    "<p>Keypoints:</p>\n",
+    "    <ul>\n",
+    "        <li>Use the pandas library to create dataframes from csv formatted data.</li>\n",
+    "    </ul>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is pandas and why do we use it?\n",
+    "\n",
+    "Pandas is a python library that is designed to work with two dimensional data arrays. It is built on numpy, another python library that specializes in numerical analysis. Numpy also has the ability to create and analyze data arrays. If you are not familiar with arrays, here are some simple examples.\n",
+    "\n",
+    "### 1D arrays\n",
+    "A one dimensional array is simply a list of items, for example, a list of the elements: H, He, Li, Be, B, etc.\n",
+    "\n",
+    "### 2D arrays\n",
```

Comment:
This explanation of 2D arrays is not quite correct.  A two dimensional array is an array which has rows and columns. It can have any number of columns and is not limited to two. I would say a two-dimensional array is data you could think of working with in a spreadsheet.

Reply:
I changed the text to read, "A two dimensional array is an array which has rows and columns. It can have any number of columns and rows. A spreadsheet with rows and columns is analogous to a 2D array."